nbs/explains/minidataapi.ipynb

```diff
@@ -0,0 +1,516 @@
+{
```

Comment:
_Commented on notebook `nbs/explains/minidataapi.ipynb` Cell 10 Line 1_
```python
# Add records to the users table
```
If we support both dicts and objects, we should show examples of both. Most examples should use objects, since that's what I think people should be using most of the time.

Reply:
Fastlite supports Dataclasses and dicts, but not regular classes.